modules/router-store/spec/integration.spec.ts

```diff
@@ -494,7 +494,7 @@ describe('integration spec', () => {
       });
   });
 
-  it('should support cancellation of initial navigation using canLoad guard', (done: any) => {
+  xit('should support cancellation of initial navigation using canLoad guard', (done: any) => {
```

Comment:
This test is failing, I don't know why yet.

Reply:
I believe this is a regression issue - https://github.com/angular/angular/issues/26284